<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PCBSDUserManager</class>
 <widget class="QDialog" name="PCBSDUserManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TrueOS User Manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pcbsdusermanager.qrc">
    <normaloff>:/add_user.png</normaloff>:/add_user.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widgetUserContainer" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>188</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushSave">
       <property name="text">
        <string>&amp;Apply</string>
       </property>
       <property name="icon">
        <iconset resource="pcbsdusermanager.qrc">
         <normaloff>:/button_ok.png</normaloff>:/button_ok.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushClose">
       <property name="text">
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="pcbsdusermanager.qrc"/>
 </resources>
 <connections/>
</ui>
